src/os/bluestore/BlueStore.cc

```diff
@@ -10305,6 +10337,11 @@ int BlueStore::get_numa_node(
   return 0;
 }
 
+void BlueStore::prepare_for_fast_shutdown()
+{
+  m_fast_shutdown = true;
```

Comment:
should we check this in queue_transaction() to be sure nothing is added to the queue after this point?